packages/module/patternfly-docs/content/design-guidelines/design-guidelines.md

```diff
@@ -9,6 +9,22 @@ id: User feedback
 source: design-guidelines
 ---
 
-Design guidelines
+Note: User feedback lives in its own package at [@patternfly/react-user-feedback](https://github.com/patternfly/react-user-feedback).
 
-COMING SOON!
+**User feedback** is a modal component that on-premises products can use to collect asynchronous feedback from users. 
+
+## Behavior
+
+When installed, user feedback may be accessed in a couple of ways: 
+
+By selecting a masthead’s help icon and choosing “Share feedback” in the dropdown menu:
+
+![Image of expanded dropdown menu with link to sharing feedback](/packages/module/patternfly-docs/content/img/dropdown-selection.png)
+
+By selecting “Share feedback” links or buttons elsewhere in the console, such as at the end of a wizard.
```

Comment:
```suggestion
Or by clicking “Share feedback” links or buttons elsewhere in the console, such as at the end of a wizard.
```

Reply:
agree! I swapped these two options into a bulleted list to help the "full sentence" flow across points, but if that's disruptive I can undo that.